ui-events/src/scroll/mod.rs

```diff
@@ -3,7 +3,7 @@
 
 use dpi::PhysicalPosition;
 
-/// Scroll delta.
+/// # Scroll delta.
```

Comment:
This seems like an error. I understand using heading levels in module docs but the headline of a type doc not so much.

Reply:
Yep, fixed in next push. Thanks.